qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -146,6 +146,7 @@ def insert_text(self, text):
         """.format(javascript.string_escape(text)))
 
     def parent(self):
```

Comment:
Since this shouldn't be part of any public API anymore now, it should be renamed to `_parent` to be "private"